packages/connectivity/connectivity/test/connectivity_test.dart

```diff
@@ -0,0 +1,92 @@
+// Copyright 2020 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'package:connectivity/connectivity.dart';
+import 'package:connectivity_platform_interface/connectivity_platform_interface.dart';
+import 'package:flutter_test/flutter_test.dart';
+import 'package:plugin_platform_interface/plugin_platform_interface.dart';
+import 'package:mockito/mockito.dart';
```

Comment:
I don't think you need mockito here at all, since you're not really mocking anything?

Reply:
Yeah I'm mocking the platform, at the bottom :)

Reply:
Yes, you're mocking the platform at the bottom, but it's returning hardcoded stuff, so you can remove the Mock class and mockito from the mix, I think :P

Reply:
(What I'm saying is you're never doing: `mockito.when(myMock.getBlah()).thenReturn("xyz")`, so no need for mockito for what you're doing now)

Reply:
But I do need it to do this: https://github.com/flutter/plugins/pull/2527/files#diff-e20d4ed7787f3ec62efa41db12110cadR64
If I don't include it, it fails

Reply:
You can't `extend Mock`, but you should be able to do `class MockConnectivityPlatform extends ConnectivityPlatform {`, and just return mocked data from all the methods there. (You can't extend Mock, but you can create a fake class that returns hardcoded data through its overridden methods)

Reply:
Discussed offline, and we'll leave this as is for now :)